src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -20,6 +20,7 @@
 abstract class Model implements Arrayable, ArrayAccess, Jsonable, JsonSerializable, QueueableEntity, UrlRoutable
 {
     use Concerns\HasAttributes,
+        Concerns\HasCasts,
```

Comment:
`HasCasts` trait contains `isDateAttribute` method which required by `HasAttributes` trait, isn't it be better to add use this trait in `HasAttributes` trait then?

Reply:
I checked that the `hasAttributes` trait is used only to extend the model, and the `isDateAttribute` method is a variable type check. That is why I took it to a logically correct place - the `hasCasts` trait.